sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -148,6 +148,19 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
     }
   }
 
+  test("verify table size calculation is accurate") {
```

Comment:
I originally meant you'd be better to test if the calculation done in parallel like https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileIndexSuite.scala#L99

Reply:
@maropu, Does this test look okay?

Reply:
We should make the title more obvious and how about `SPARK-24626 parallelize location size calculation in Analyze Table command`?